util/generate_testrun_index.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from google.cloud import storage
+
+"""
+Scans all WPT results directories and generates an index.
+
+You must be logged into gcloud to run this script. The non-public permission
```

Comment:
1. Is this permission something that the wptdashboard project owner has to grant?
2. Would it also sound OK to say "The permission..." instead of "The non-public permission..."?